tests/notus/test_notus.py

```diff
@@ -249,9 +249,9 @@ def test_check_field_names_lsc_unordered(self):
             "CREATION_DATE",
             "LAST_MODIFICATION",
             "SOURCE_PKGS",
-            "ADVISORY_ID",
-            "CVSS_BASE_VECTOR",
-            "CVSS_BASE",
+            "ADVISORY_ID" "SEVERITY_ORIGIN",
```

Comment:
Was this intentional or shouldn't the tags be on 2 different lines/entries?

Other than that, everything looks good to me.

Reply:
Oops!. Fixed, thanks